types/kiosk-browser.d.ts

```diff
@@ -0,0 +1,88 @@
+declare namespace KioskBrowser {
```

Comment:
I assume most of this file is copy/pasta?

Reply:
Yeah this is the kiosk types file from election manager